### PR TITLE
Fix v20 php8 patch002

### PR DESCRIPTION
**Pull request by thersane-john**
# FIX
 
Php errors

### CODE REVIEW

File: htdocs/core/modules/expedition/mod_expedition_ribera.php [hunk 1]
```diff
@@ -98,13 +98,21 @@ public function info($langs)
...
 		$mysoc->code_client = 'CCCCCCCCCC';
 		$mysoc->typent_code = 'TTTTTTTTTT';
-		$numExample = $this->getNextValue($mysoc, '');
+
+
+		if (!class_exists('Expedition')) {
+			require_once DOL_DOCUMENT_ROOT.'/expedition/class/expedition.class.php';
+		}
+
+		$expedition = new Expedition($db);
```
Comment (eldy) [inline htdocs/core/modules/expedition/mod_expedition_ribera.php:113]: Why not using $this->db ?

**Comment by thersane-john** [comment]
> [PR not sync and branch locked]

I know but I haven't the checkbox for **Allow edits from maintainers.**

**Comment by frederic34** [comment]
you don't have this :
image

**Comment by thersane-john** [comment]
> you don't have this : image

Nop, I have it on my @atm-john 
user account but  not on this one...

**Comment by frederic34** [comment]
https://github.com/orgs/community/discussions/5634

**Comment by thersane-john** [comment]
Thanks @frederic34 now I know why ;-)

**Pull request closed without merging**